### ISSUE TITLE
Permission denied in "other detected applications" section

### ISSUE THREAD

**Issue by Lethalchip** [Fmstrat/winapps#298]
Hello! 

I have encountered an issue when trying to get Adobe programs to run. I am using Creative Cloud on Windows 10 (the vm), and im on Linux pop-os 5.15.5.

I noticed that none of the Adobe programs are recognized in the `pre-configured applications` section, which isn't a big deal because they're in the `other detected applications` section. That said, when I select *any* app from the "other detected applications" section, I am met with: `/usr/local/share/winapps/apps/<APP HERE>/info: Permission denied`. 

I find this strange, because I am entering the correct sudo password at the beginning of the script? 

One other note is that I am running this system-wide, not user based. If I run the install script and pick user, even the pre-configured applications get this "Permission denied" error. 

I tried manually forcing permissions by chmod'ing the entire `/usr/local/share/winapps/apps` folder (temporarily), but no dice there.

Please let me know what I can do! 

Screenshot:
![issue](https://user-images.githubusercontent.com/36650721/145889100-cb92058a-346c-45b6-b076-4642b627d4a2.png)

**Comment by Lethalchip** [comment]
It's also worth noting that everything runs perfectly for the `pre-configured applications`.

**Comment by Benmac83** [comment]
Have you tried "chown" on that folder? No idea if it will work but just a question

**Comment by Benmac83** [comment]
sudo chown scott:scott -R /usr/local/share/winapps/apps

(I took your username from the screenshot, if it's wrong just change it

I am not responsible if anything goes wrong but this shouldn't break anything. And should *maybe* fix the issue)

**Comment by Lethalchip** [comment]
> sudo chown scott:scott -R /usr/local/share/winapps/apps
> 
> (I took your username from the screenshot, if it's wrong just change it
> 
> I am not responsible if anything goes wrong but this shouldn't break anything. And should _maybe_ fix the issue)

Hi Benmac, thank you for the suggestion! I just tried it.

Unfortunately, it didn't work out. Seems like the Installer.sh script does some ownership stuff, because the chown was overwritten when I ran it- it's back to root:root instead of scott:scott, and I'm still getting the issue.

I did more digging, and the photoshop, illustrator, and acrobat folders are actually.. empty! All of the apps in the `pre-configured applications` section have an icon.svg and an info file, but the three apps that are not in the `other detected applications` do not have those files. 

Because of this, I'm now assuming it's not really a permissions thing, and moreso some issue with the script that fetches the icon and info from the VM?

**Comment by Benmac83** [comment]
Can you actually launch the Adobe apps? On my system if I run them from terminal (to get a debug) it states it needs me to run “install.sh”

**Comment by Lethalchip** [comment]
> Can you actually launch the Adobe apps? On my system if I run them from terminal (to get a debug) it states it needs me to run “install.sh”

Yes, I can run the applications within the VM, so it's not the end of the world. Work has been crazy so I haven't had much time to troubleshoot. Appreciate your suggestions thus far!

**Comment by alexfeed1990** [comment]
> Can you actually launch the Adobe apps? On my system if I run them from terminal (to get a debug) it states it needs me to run “install.sh”

yea same for me, even if i do a fresh install like issue #308 on the user, the other apps dont work like adobe premiere or paint, if i do it on the system it still gives me the permission denied. if i install it on the user it does not do that. any help??

**Comment by alexfeed1990** [comment]
nevermind, i got it working, just do a fresh install on user with the chmod winapps/apps folder. premiere wont launch tho, and if you launch something like after effects, you need to wait a while then launch something like cmd and only then you can use after effects, etc..
